lib/Doctrine/Migrations/Tools/Console/Command/StatusCommand.php

```diff
@@ -47,7 +47,7 @@ protected function configure() : void
         parent::configure();
     }
 
-    public function execute(InputInterface $input, OutputInterface $output) : void
+    public function execute(InputInterface $input, OutputInterface $output) : ?int
```

Comment:
The return types shouldn't be nullable, you never return null.
Also if upstream ever adds `?int` return type, removing nullability is valid covariant change.

Reply:
I made it match what the Symfony console doc block has currently. You are correct that in our commands we never return null, so we could change this here.